examples/02_decoding/plot_haxby_frem.py

```diff
@@ -42,53 +46,32 @@
 background_img = mean_img(func_filenames)
 
 ##############################################################################
-# Fit SpaceNet with a Graph-Net penalty
+# Fit fREM
 # --------------------------------------
-from nilearn.decoding import SpaceNetClassifier
+from nilearn.decoding import fREMClassifier
 
+decoder = fREMClassifier('svc', clustering_percentile=10,
+                         screening_percentile=10, cv=10)
 # Fit model on train data and predict on test data
-decoder = SpaceNetClassifier(memory="nilearn_cache", penalty='graph-net')
 decoder.fit(X_train, y_train)
 y_pred = decoder.predict(X_test)
 accuracy = (y_pred == y_test).mean() * 100.
-print("Graph-net classification accuracy : %g%%" % accuracy)
+print("fREM classification accuracy : %g%%" % accuracy)
 
 #############################################################################
-# Visualization of Graph-net weights
+# Visualization of fREM weights
 # ------------------------------------
 from nilearn.plotting import plot_stat_map, show
-coef_img = decoder.coef_img_
-plot_stat_map(coef_img, background_img,
-              title="graph-net: accuracy %g%%" % accuracy,
+%matplotlib inline
+plot_stat_map(decoder.coef_img_["face"], background_img,
+              title="fREM: accuracy %g%%, 'face coefs'" % accuracy,
               cut_coords=(-52, -5), display_mode="yz")
-
-# Save the coefficients to a nifti file
-coef_img.to_filename('haxby_graph-net_weights.nii')
-
-
-##############################################################################
-# Now Fit SpaceNet with a TV-l1 penalty
-# --------------------------------------
-decoder = SpaceNetClassifier(memory="nilearn_cache", penalty='tv-l1')
-decoder.fit(X_train, y_train)
-y_pred = decoder.predict(X_test)
-accuracy = (y_pred == y_test).mean() * 100.
-print("TV-l1 classification accuracy : %g%%" % accuracy)
+coef_img.to_filename('haxby_fREM_weights.nii')
```

Comment:
```suggestion
coef_img.to_filename('haxby_fREM_weights.nii.gz')
```

Reply:
Or, should this go to some output directory ? (I haven't checked what other examples do in that respect)

Reply:
we don't need to save it at all, this line can just be removed